pkg/controller/masteruserrecord/masteruserrecord_controller.go

```diff
@@ -276,45 +279,61 @@ func (r *ReconcileMasterUserRecord) useExistingConditionOfType(condType toolchai
 	}
 }
 
-func (r *ReconcileMasterUserRecord) manageCleanUp(logger logr.Logger, mur *toolchainv1alpha1.MasterUserRecord) error {
+func (r *ReconcileMasterUserRecord) manageCleanUp(logger logr.Logger, mur *toolchainv1alpha1.MasterUserRecord) (time.Duration, error) {
 	for _, ua := range mur.Spec.UserAccounts {
-		if err := r.deleteUserAccount(logger, ua.TargetCluster, mur.Name); err != nil {
-			return r.wrapErrorWithStatusUpdate(logger, mur, r.setStatusFailed(toolchainv1alpha1.MasterUserRecordUnableToDeleteUserAccountsReason), err,
+		requeueTime, err := r.deleteUserAccount(logger, ua.TargetCluster, mur.Name)
+		if err != nil {
+			logger.Error(err, "unable to clean up MasterUserRecord as part of deletion")
```

Comment:
This error is logged in reconcile() (line 132), correct? We do not have to duplicate it here.

Reply:
yeah, I would remove this one and keep only the one at line 132

Reply:
Nice catch! I missed that

Reply:
Done in eefde512ed3508572559a09206194139afb80df6